docs/bigquery/snippets.py

```diff
@@ -338,6 +363,40 @@ def test_get_table(client, to_delete):
     # [END get_table]
 
 
+def test_show_table(client, to_delete):
+    """Show a table's properties"""
+    DATASET_ID = 'show_table_dataset_{}'.format(_millis())
+    TABLE_ID = 'show_table_table_{}'.format(_millis())
+    dataset_ref = client.dataset(DATASET_ID)
+    dataset = bigquery.Dataset(dataset_ref)
+    client.create_dataset(dataset)
+    to_delete.append(dataset)
+
+    table = bigquery.Table(dataset.table(TABLE_ID), schema=SCHEMA)
+    table.description = ORIGINAL_DESCRIPTION
+    table = client.create_table(table)
+    to_delete.insert(0, table)
+
+    # [START bigquery_show_table]
```

Comment:
Show table? Which section in the docs does this correspond to? I'd think that `bigquery_get_table` might better match the convention used for other region tags.

If you think that conflicts too much with the `get_table` above, I think that one this is more like `bigquery_get_table_reload` since it refreshes a table they've already fetched, but I can't imagine where we'd use in the docs (other than in the migration guide for the folks used to "reload")

Reply:
It's for this [section](https://cloud.google.com/bigquery/docs/tables#getting_table_information).  I can update since adding `bigquery_` to the front will distinguish it. It looks like sample labeled `get_table` is just in the migration guide and `usage.rst`.